Makefile

```diff
@@ -135,13 +135,12 @@ golangci: error
 
 ## Build Meshery's protobufs (requires go$(GOVERSION)).
 proto-build:
+	# see https://developers.google.com/protocol-buffers/docs/reference/go-generated
 	# see https://grpc.io/docs/languages/go/quickstart/
-	# go$(GOVERSION) get -u google.golang.org/grpc
-	# go$(GOVERSION) get -u google.golang.org/protobuf/cmd/protoc-gen-go \
-	#         google.golang.org/grpc/cmd/protoc-gen-go-grpc
-	# PATH=$(PATH):`pwd`/../protoc/bin:$(GOPATH)/bin
-	# export PATH=$PATH:`pwd`/../protoc/bin:$GOPATH/bin
-	protoc -I meshes/ meshes/meshops.proto --go-grpc_out=./server/meshes/ --go_out=./server/meshes/
+	export PATH=$(PATH):$(GOBIN)
+	go$(GOVERSION) install google.golang.org/protobuf/cmd/protoc-gen-go@latest
+	go$(GOVERSION) install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.2
```

Comment:
@acald-creator what do you think: go for `latest` or pin to `v1.2`?

Reply:
Agreed @leecalcote only because `protoc-gen-go` is being pulled at the `latest` version, `protoc-gen-go-grpc` can also be changed to pull at `latest` as well.

If there are any blockers, we can address it and pin it if need be.

@abdullah1308 Can you make one last change on this line to use `latest` instead of `v1.2`

Reply:
@acald-creator Done